internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -317,10 +316,8 @@ func (r *ISBServiceRolloutReconciler) reconcile(ctx context.Context, isbServiceR
 			requeueDelay = min(requeueDelay, common.DefaultRequeueDelay)
 		}
 	}
+	r.updateProgressiveResultsMetrics(isbServiceRollout, newISBServiceDef, true)
```

Comment:
It seems strange to put it here if it's for progressive metrics, right? After all, this code is hit even if we're in say PPND strategy.